Lib/glyphsLib/classes.py

```diff
@@ -1234,8 +1239,20 @@ def shouldWriteValueForKey(self, key):
         if key in ("xHeight", "capHeight", "ascender"):
             # Always write those values
             return True
+        if key == "name":
+            if self.font and int(self.font.appVersion) >= 1075:
+                return True
+            return False
         return super(GSFontMaster, self).shouldWriteValueForKey(key)
 
+    @property
```

Comment:
In Glyphs 1088, `font` is a regular method, not a property (the property access returns an "objective C native selector")

Reply:
what do you mean a "method"? A callable as in `master.font()`?
we shouldn't expose or try to mimic the the objc interface here

Reply:
Yes a callable:
```python
>>> print(Glyphs.font.masters[0].font().masters[0])
<GSFontMaster "Regular" width 100.0 weight 100.0>
>>> print(Glyphs.font.masters[0].font.masters[0])
Traceback (most recent call last):
  File "<string>", line 5, in <module>
AttributeError: 'objc.native_selector' object has no attribute 'masters'
```

I agree that it would be much better to have a method, but then the scripts written for glyphsLib would not work in Glyphs.app (is that still a goal?)